app/main.py

```diff
@@ -1,3 +1,63 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km: float = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union[Distance, int, float]) -> Distance:
```

Comment:
You could omit using `Union` here and in other methods as well and replace it with `|`, e.g.:
```suggestion
    def __add__(self, other: Distance | int | float) -> Distance:
```